### DISCUSSION TITLE
Nuke all ecto-ASCII characters

### DISCUSSION THREAD

**Discussion by adlai** [community/community#48607] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

GitHub is ~terrible~ _difficult_, and I will do my best to phrase this complaint as request for help, instead of simply bitching about how social networks are literally brain cancer in the Mind of God.

Does anyone recommend some simple solution to the perfusion of emojis all over the website? It's distracting, and sometimes worse than merely obnoxious. Obviously being a cynical and ungrateful keyboard jockey does not constitute a handicap, so I've not filed this under Accessibility, although I can imagine that some folks using and/or developing screenreaders might share my distaste for textual artistry.

**Comment by sbhask** [comment]
[this works](https://chrome.google.com/webstore/detail/emoji-blocker/eimknojlmcfpmofldcdiaddfimjchmcj/related?hl=en-GB)

**Reply by sbhask** [reply]
and quite well actually

**Reply by adlai** [reply]
thank you for the link; any idea whether morally upstanding thieves have ported the damn crank to any other wall-kept garden?

[it'd be absolutely farthing awesome if the list of obnoxious codepages were configurable, although let's start from not living and dying within one open source project]

**Reply by sbhask** [reply]
not sure! can you mark my answer as correct - i'm on a badge crusade lol

**Reply by adlai** [reply]
> not sure! can you mark my answer as correct - i'm on a badge crusade lol

not yet; and good luck with your campaigning.

**Reply by adlai** [reply]
> not sure! can you mark my answer as correct - i'm on a badge crusade lol

Ideally, the solution would either be a supported mode within the site settings; or one of the CLI tools officially acknowledged by the site's administration, even if not affiliated or maintained by the same organisation.
